tasks/deploy/config/linea/factory.ts

```diff
@@ -0,0 +1,74 @@
+import { Address, parseEther, zeroAddress } from "viem";
+import { HyperdriveFactoryConfig, parseDuration } from "../../lib";
+
+// The name of the factory.
+export const LINEA_FACTORY_NAME = "ElementDAO Hyperdrive Factory";
+
+// The name of the forwarder factory.
+export const LINEA_FACTORY_FORWARDER_NAME =
+    "ElementDAO ERC20 Factory Forwarder";
+
+export const LINEA_FACTORY: HyperdriveFactoryConfig = {
+    name: LINEA_FACTORY_NAME,
+    prepare: async (hre, options) => {
+        await hre.hyperdriveDeploy.ensureDeployed(
+            LINEA_FACTORY_FORWARDER_NAME,
+            "ERC20ForwarderFactory",
+            [LINEA_FACTORY_FORWARDER_NAME],
+            options,
+        );
+    },
+    constructorArguments: async (hre) => [
+        {
+            governance: (await hre.getNamedAccounts())["deployer"] as Address,
+            deployerCoordinatorManager: (await hre.getNamedAccounts())[
+                "deployer"
+            ] as Address,
+            hyperdriveGovernance: (await hre.getNamedAccounts())[
+                "deployer"
+            ] as Address,
+            defaultPausers: [
+                (await hre.getNamedAccounts())["deployer"] as Address,
+                (await hre.getNamedAccounts())["pauser"] as Address,
+            ],
+            feeCollector: zeroAddress,
+            sweepCollector: zeroAddress,
+            checkpointRewarder: zeroAddress,
+            checkpointDurationResolution: parseDuration("1 hours"),
+            minCheckpointDuration: parseDuration("24 hours"),
+            maxCheckpointDuration: parseDuration("24 hours"),
+            minPositionDuration: parseDuration("7 days"),
+            maxPositionDuration: parseDuration("730 days"),
+            minFixedAPR: parseEther("0.005"),
+            maxFixedAPR: parseEther("0.1"),
+            minTimeStretchAPR: parseEther("0.005"),
+            maxTimeStretchAPR: parseEther("0.1"),
+            minCircuitBreakerDelta: parseEther("0.01"),
+            maxCircuitBreakerDelta: parseEther("0.2"),
```

Comment:
this seems low but I could be totally wrong

Reply:
This is, if anything, a bit high. We left it this high for flexibility